<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HapusDataNomorDiMaster</class>
 <widget class="QDialog" name="HapusDataNomorDiMaster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hapus Data Nomor di Data Master</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Hapus Data Nomor di Data Master</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="3" column="0" colspan="3">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Format csv nomor yang sah:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="lineEdit_upload_nomor">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="clearButtonEnabled">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QCheckBox" name="checkBox_is_use_header_csv">
            <property name="text">
             <string>Menggunakan header csv</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QPushButton" name="pushButton_jalankan_hapus_nomor">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Jalankan Hapus Nomor</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="pushButton_upload_nomor_hapus">
            <property name="text">
             <string>Upload data nomor</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Upload Nomor</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>   header nomor(opsional)
   angka_nomor</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QPushButton" name="pushbutton_tidak_jadi">
            <property name="text">
             <string>Tidak Jadi</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="3">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Tanpa proses identifikasi cek duplikasi nomor</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
